lib/commons/dom/focus-disabled.js

```diff
@@ -2,6 +2,13 @@ import AbstractVirtualNode from '../../core/base/virtual-node/abstract-virtual-n
 import { getNodeFromTree } from '../../core/utils';
 import isHiddenWithCSS from './is-hidden-with-css';
 
+function isDisabledAttrAllowed(nodeName) {
+  // Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled 
+  const allowedNodeNames = new Set(["button", "command", "fieldset", "keygen", "optgroup", 
+      "option", "select", "textarea", "input"]);
+  return allowedNodeNames.has(nodeName);
```

Comment:
Thanks for listing the source, that was very helpful.

Since the `allowedNodeNames` never changes, we should move the creation of the array outside of the function so it's not recreated every time the function is called. Also `Set` is a bit much and we can just make do with an array using `.includes`.

```suggestion
// Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled 
const allowedDisableNodeName = [
  'button', 
  'command', 
  'fieldset', 
  'keygen', 
  'optgroup', 
  'option', 
  'select', 
  'textarea', 
  'input'
];

function isDisabledAttrAllowed(nodeName) {
  return allowedDisableNodeName.includes(nodeName);
}
```